commands/command_pre_push.go

```diff
@@ -83,8 +100,34 @@ func prePushCommand(cmd *cobra.Command, args []string) {
 			Print("Error scanning for Git LFS files in %q", left)
 			ExitWithError(err)
 		}
+
+		for _, p := range pointers {
+			if l, ok := lockSet[p.Name]; ok {
+				if l.Name == name && l.Email == email {
+					myLocks = append(myLocks, l.Path)
+				} else {
+					lockConflicts = append(lockConflicts, p.Name)
+				}
+			}
+		}
+
+		if len(lockConflicts) > 0 {
+			Error("Some files are locked in %s...%s", left, cfg.CurrentRemote)
```

Comment:
Maybe we could be explicit like "You cannot push changes because someone else has these files locked"